apis/pipeline/pipeline.yaml

```diff
@@ -8057,6 +8072,21 @@ components:
                 secretId:
                     type: string
                     example: "62bc3c75-91fb-4670-bad4-24b401a9deac"
+                location:
+                    type: string
+                    example: "us-east"
+                useClusterSecret:
+                    type: boolean
+                    example: false
+                    description: "relevant only in case of Amazon clusters. By default set to false in which case you must add snapshot permissions to your instance profile. Should you set to true Pipeline will deploy your cluster secret to the cluster."
+                storageAccount:
+                    type: string
+                    example: "my-storage-account"
+                    description: "required only case of Azure"
+                resourceGroup:
+                    type: string
+                    example: "my-resource-group"
+                    description: "required only case of Azure"
```

Comment:
**Note**: these may be a little cleaner to be defined in a oneof structure where the Amazon and EKS only values are defined in their respective schemas.

This approach may be worth considering for `CreateBackupBucketRequest` as well.